manifests/c/CiderCollective/Cider/1.4.3.1793/CiderCollective.Cider.installer.yaml

```diff
@@ -0,0 +1,16 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: CiderCollective.Cider
+PackageVersion: 1.4.3.1793
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+UpgradeBehavior: install
+ReleaseDate: 2022-04-18
+ElevationRequirement: elevationRequired
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/ciderapp/cider-releases/releases/download/v1.4.3.1793/Cider-Setup-winget-1.4.3-alpha.1793.exe
```

Comment:
Another alpha, is this the correct repo?

> This repository manages and archives Cider builds made with Circle CI
> NOTE: These builds are experimental and have NOT been tested.

Reply:
Yes, I have also verified the update condition and it's correct, but not getting why a PR is still being made.